components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.geo-dashboard/public/js/geo_fencing.js

```diff
@@ -39,30 +39,30 @@ function loadGeoFencing() {
 
 function openTools(id) {
     lastId = id;
-	if (drawControl) {
-		try{
-    	map.removeControl(drawControl);
-    	} catch(e) {
-    		console.log("error: " + e.message);
-    	}
-		console.log("removed drawControl");
+    if (drawControl) {
```

Comment:
remove console.log lines from this file

Reply:
Removed all except the ones inside catch blocks. c653051